x-pack/plugin/esql/qa/testFixtures/src/main/resources/stats.csv-spec

```diff
@@ -1552,6 +1576,54 @@ s2point1:d | s_mv:i | languages:i
 2.1        | 3      | null
 ;
 
+medianOfConst#[skip:-8.13.99,reason:supported in 8.14]
+FROM employees
+| STATS s1 = median(1), s_mv = median([-1, 0, 1, 3]), s_null = median(null)
+;
+
+s1:d | s_mv:d | s_null:d
+1.0  | 0.5    | null
+;
+
+medianOfConstGrouped#[skip:-8.13.99,reason:supported in 8.14]
+FROM employees
+| STATS s2point1 = median(2.1), s_mv = median([-1, 0, 1, 3]) by languages
+| SORT languages
+;
+
+s2point1:d | s_mv:d | languages:i
+2.1        | 0.5    | 1
+2.1        | 0.5    | 2
+2.1        | 0.5    | 3
+2.1        | 0.5    | 4
+2.1        | 0.5    | 5
+2.1        | 0.5    | null
+;
+
+countDistinctOfConst#[skip:-8.13.99,reason:supported in 8.14]
+FROM employees
+| STATS s1 = count_distinct(1), s_mv = count_distinct([-1, 0, 3, 1, -1, 3]), s_null = count_distinct(null), s_param = count_distinct([-1, 0, 3, 1, -1, 3], 5) 
+;
+
+s1:l | s_mv:l | s_null:l | s_param:l
+1    | 4      | 0        | 4
```

Comment:
s_mv = s_param looks weird.

Reply:
> s_mv = s_param looks weird.

The second/optional argument(precision), tricked me,  before knowing the count_distinct is approximate count distinct. It is irrelevant to the issue this PR addresses though. 

Some databases offers two ways to count distinct, perhaps we can consider later:
count(distinct <column>): this is an exact count, which may perform slow on large dataset
approximate_count_distinct function: this is an estimation.